include/cuco/static_map.cuh

```diff
@@ -946,12 +944,38 @@ class static_map {
     /**
      * @brief Inserts the specified key/value pair into the map.
      *
-     * Returns a pair consisting of an iterator to the inserted element (or to
-     * the element that prevented the insertion) and a `bool` denoting whether
-     * the insertion took place. Uses the CUDA Cooperative Groups API to
-     * to leverage multiple threads to perform a single insert. This provides a
-     * significant boost in throughput compared to the non Cooperative Group
-     * `insert` at moderate to high load factors.
+     * Returns a fetch_insert_result, a pair consisting of an iterator to the
+     * inserted element (or to the element that prevented the insertion) and
+     * a `bool` denoting whether the insertion took place.
+     *
+     * Note: This function is slower than insert for non-packable key/value
+     * types because it has to ensure a valid iterator in case of duplicates.
+     * For the same reason, the function must not be called concurrently with
+     * insert.
+     *
```

Comment:
```suggestion
     *
     * Concurrent `fetch_insert`/`find` is supported if `supports_concurrent_insert_find()` returns true.
     *  
```

Reply:
I've added the note to the previous paragraph.